src/coreclr/interop/comwrappers.hpp

```diff
@@ -128,6 +59,11 @@ class ManagedObjectWrapper
     // Check if the wrapper has been marked to be destroyed.
     bool IsMarkedToDestroy() const;
 
+    InteropLib::OBJECTHANDLE GetTarget() const
+    {
+        return target;
+    }
```

Comment:
I get placing this in the header, but please move this to the cpp unless there is a compelling reason to do this here.

Reply:
I can move the impl to the cpp.